tests/node_eviction_test.go

```diff
@@ -0,0 +1,238 @@
+package tests_test
+
+import (
+	"flag"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+var _ = Describe("Node Eviction", func() {
+
+	flag.Parse()
+
+	var (
+		simpleVmi   tests.VmManifest
+		vmBackedVmi tests.VmManifest
+		rsBackedVmi tests.VmManifest
+	)
+
+	ns := ktests.NamespaceTestDefault
+
+	BeforeEach(func() {
+		ktests.BeforeTestCleanup()
+
+		args := []string{"get", "node", "--selector=node-role.kubernetes.io/compute=true", "--output=jsonpath={.items..metadata.name}"}
+		output, err := ktests.RunOcCommand(args...)
+		Expect(err).ToNot(HaveOccurred())
+
+		var count int
+		nodes := strings.Fields(output)
+		for _, node := range nodes {
+			args := []string{"get", "node", node}
+			output, err := ktests.RunOcCommand(args...)
+			Expect(err).ToNot(HaveOccurred())
+
+			if !strings.Contains(output, "SchedulingDisabled") {
+				count += 1
+			}
+		}
+		if count < 2 {
```

Comment:
so to make sure this tests could be executed successfully, we have to make sure at least two nodes are `Ready` is that right ? why dont we check this condition?

Reply:
`cnv-executor-gouyang-node2.example.com     Ready,SchedulingDisabled` tells why.